### DISCUSSION TITLE
Add optional commit @authors to get Contributors section

### DISCUSSION THREAD

**Discussion by bsojka** [metcalfc/changelog-generator#265] [Ideas]
Hi! Thank you for this handy action. I’ve seen it implemented in various projects and decided to use it as well — it’s great 👍🏼.

One thought that came to my mind when preparing my first release:
It would be great if you could add an optional boolean input, `authors` or `mentions` (`false` by default for compatibility). When set to `true`, it would include each commit’s author in a mention format (e.g. „… by @metcalfc (#0)”), similar to the new built-in release notes auto-generator. This would create an automatic **Contributors** section in the release, which is clear, sleek, and helpful.

For a quick comparison, feel free to check out [my example](https://github.com/Inetum-Poland/bootstrap-buddy/releases), where a Pre–release is auto–generated, while in the latest Release, I manually added („faked”) mentions to display a **Contributors** section.

**Comment by metcalfc** [comment]
@bsojka I agree this looks useful. Do you have this working in a branch?

**Reply by bsojka** [reply]
Unfortunately, I don’t have a branch for this, @metcalfc. To be honest, I’m still relatively new to Actions and lack the expertise to implement it myself. Do you think this is something you might have the time and interest to develop in the foreseeable future?
